blog/en/blog/2023/08/11/rate-limit.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Rate Limit Your APIs With Apache APISIX
+authors:
+  - name: Navendu Pottekkat
+    title: Author
+    url: https://github.com/navendu-pottekkat
+    image_url: https://avatars.githubusercontent.com/u/49474499
+keywords:
+  - Rate limit
+  - APISIX
+  - Redis
+  - Traffic control
+description: A guide to using the rate limit plugins in Apache APISIX with some practical examples.
+tags: [Plugins]
+image: https://static.apiseven.com/uploads/2023/08/07/jIphRpzX_rate-limit.png
+---
+
+> In this article, we will look at examples of how we can use the rate limiting plugins in APISIX.
+
+<!--truncate-->
+
+<head>
+    <link rel="canonical" href="https://navendu.me/posts/rate-limit/" />
+</head>
+
+Setting up rate limits is a solid way to improve the reliability of your services.
+
+You can ensure your services are not overloaded by setting up rate limits, providing consistent performance for the end users. It can also help enhance security by preventing denial-of-service (DoS) attacks which can take down your services.
+
+Rate limiting can also be part of your business requirement where you want separate quotas for different tiers of users. For example, you can have a free version of your API where users can only make a small number of API calls and a paid version that allows ample API usage.
+
+If you are using [Apache APISIX](https://apisix.apache.org) as your API gateway, you can leverage the rate limiting plugins, [limit-req](https://apisix.apache.org/docs/apisix/plugins/limit-req/), [limit-conn](https://apisix.apache.org/docs/apisix/plugins/limit-conn/), and [limit-count](https://apisix.apache.org/docs/apisix/plugins/limit-count/) to achieve this.
+
+You can always set this up in your services directly without configuring it in APISIX. But as the number of your services increases, with each service having different constraints, setting up and managing different rate limits and updating them might become difficult.
```

Comment:
"difficult" is not precise enough. I'd mention the pain of going to each service and configuring them: Ops need a central point of configuration